### PR TITLE
fix: Update binary names upgrade path testing

### PR DESCRIPTION
**Pull request by gzukel**
# Description

Fixing binary names for upgrade path testing after the change.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Via Github Action Pipeline.

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [x] Tested via GitHub Actions 

# Checklist:

- [x] I have added unit tests that prove my fix feature works

### CODE REVIEW

File: .github/workflows/rc-release.yml [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+          generate_release_notes: true
+
+      - name: Publish Release Files
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT_GITHUB_SERVICE_ACCT }}
+        run: |
+          touch .release-env
+          make release
+
+      ## TODO - Will add in later after optimizing docker images
```
Comment (lumtis) [inline .github/workflows/rc-release.yml:78]: Maybe we can create an issue for this?

File: .github/workflows/rc-release.yml [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+      #   with:
+      #     discord_token: "${{ secrets.DISCORD_TOKEN }}"
+      #     discord_channel_id: "${{ secrets.DISCORD_CHANNEL_ID }}"
+      #     discord_message: |
+      #       Hey <@&1122981184255840306>! A new version of the zetachain node has been released.
+
+      #       Minor Version Upgrade (e.g. v5.0.x to v5.1.x) can be applied without a governance proposal.
+      #       Please review the release notes for any specific upgrade instructions or considerations.
+
+      #       See the release notes for more details. https://github.com/zeta-chain/node/releases/tag/${{ env.BINARY_VERSION }}
```
Comment (lumtis) [inline .github/workflows/rc-release.yml:228]: Does it make sense of keeping it commented?

**Review by lumtis — APPROVED** [review]
LGTM, need a review from DevOps.
Also, I think it's best to keep all TODOs tracked with Giothub issues.

File: .github/workflows/rc-release.yml (hunk 2 above)
Comment (CharlieMc0) [reply .github/workflows/rc-release.yml:228]: Once we get our release process a little more refined we may want to notify discord for minor version too so we can leave it in for now

**Review by CharlieMc0 — APPROVED** [review]

**Pull request merged**